lib/mix/tasks/sbom.cyclonedx.ex

```diff
@@ -30,7 +33,10 @@ defmodule Mix.Tasks.Sbom.Cyclonedx do
       files for each application, rather than a single file for the entire
       project
     * `--schema` (`-s`): schema version to be used, defaults to
-      "#{@default_schema}".
+      "#{@default_schema}"
+    * `--format` (`-t`): output format: xml (default) or json
```

Comment:
DX suggestion: when an output file is supplied, you could detect the format from the file extension